profile/ffsn.py

```diff
@@ -0,0 +1,80 @@
+import time
+import numpy as np
+from pyffs import ffsn_sample, ffsn, ffsn_comp
+from pyffs.func import dirichlet_2D
+import matplotlib.pyplot as plt
+import click
```

Comment:
Is this a new dependency? Didn't see it in the requirements file.

Reply:
And not sure we want it as a necessary dependency. Only required for testing.

Reply:
Is it really a dependency if it's a test script though? For people installing the package, e.g. via `pip`, they won't need `pytest`, `click`, and `matplotlib`.

Reply:
ok